### ISSUE TITLE
LHN option and badge

### ISSUE THREAD

**Issue by madmax330** [Expensify/App#45181]
# Part of the Invoicing payments project
Main issue: https://github.com/Expensify/Expensify/issues/382977
Doc section: https://docs.google.com/document/d/1CF5gw-SJx7ckZCgTJNj1IqBnsRuSxD2Bfd_uB5rRGxk/edit#heading=h.yv8t9d8vscvw
Project: https://github.com/orgs/Expensify/projects/114/views/3

# Feature Description

# Manual Test Steps

# Automated Tests

**Comment by VickyStash** [comment]
Hi, I'm Viktoryia from Callstack - expert contributor group - and I would like to work on this issue.

**Comment by VickyStash** [comment]
**Note:** Later phase of implementation

**Comment by madmax330** [comment]
This is waiting on some backend and front-end before it can be started

**Comment by rezkiy37** [comment]
Hi, I am Michael (Mykhailo) from Callstack, an expert agency and I can work on this issue.

**Comment by madmax330** [comment]
@dylanexpensify can you just make the payment for @getusha for the PR Review here?

**Comment by dylanexpensify** [comment]
On it!

Payment summary:

Contributor+: @getusha $250 

Please apply/request!

**Comment by garrettmknight** [comment]
$250 approved for @getusha
